tests/delete/tests.py

```diff
@@ -263,6 +264,106 @@ def test_restrict_gfk_no_fast_delete(self):
         self.assertFalse(GenericDeleteBottom.objects.exists())
 
 
+@skipUnlessDBFeature("supports_foreign_keys")
+class OnDeleteDbTests(TestCase):
+
+    def test_db_cascade(self):
+        a = BaseDbCascade.objects.create()
+        b = RelToBaseDbCascade.objects.create(
+            name='db_cascade',
+            db_cascade=a
+        )
+        b.db_cascade.delete()
+        # BaseDbCascade is deleted
+        self.assertFalse(
+            BaseDbCascade.objects.filter(pk=a.pk).exists()
+        )
+        # RelToBaseDbCascade is cascade deleted
+        self.assertFalse(
+            RelToBaseDbCascade.objects.filter(name="db_cascade").exists()
+        )
+
+    def test_db_cascade_query_count(self):
+        """
+        A models.DB_CASCADE relation doesn't trigger a query
+        """
```

Comment:
This docstring can be collapsed to a single line. Please add a period to the end too.